gcpStandalone/src/index.ts

```diff
@@ -0,0 +1,34 @@
+import * as ff from '@google-cloud/functions-framework';
+// import {getCompositions} from '@remotion/renderer';
+import {getBrowserInstance} from './helpers/get-browser-instance';
+import { existsSync, readdirSync } from 'node:fs';
+
+
+ff.http('TypescriptFunction', async (req: ff.Request, res: ff.Response) => {
+	const AWSUrl =
+		'https://remotionlambda-11kow3vq6f.s3.us-east-1.amazonaws.com/sites/xmycbufjs3/index.html';
+	const GCPUrl =
```

Comment:
When you bundle the site, you need to pass `publicPath` as `/remotionlambda-test/sites/xmycbufjs3/`, otherwise the bundle.js will be loaded from the root of the domain